src/Stratis.Bitcoin.Features.Wallet/Wallet.cs

```diff
@@ -726,7 +728,25 @@ public IEnumerable<UnspentOutputReference> GetSpendableTransactions(int currentC
                     if (transactionData.BlockHeight != null)
                         confirmationCount = countFrom >= transactionData.BlockHeight ? countFrom - transactionData.BlockHeight : 0;
 
-                    if (confirmationCount >= confirmations)
+                    if (confirmationCount < confirmations)
+                        continue;
+
+                    bool isCoinBase = transactionData.IsCoinBase ?? false;
+                    bool isCoinStake = transactionData.IsCoinStake ?? false;
+
+                    // This output can unconditionally be included in the results.
+                    if (!isCoinBase && !isCoinStake)
```

Comment:
Maybe we can combine the 2 ifs if we format it like this:
```
if ((!isCoinBase && !isCoinStake) ||
	((isCoinBase || isCoinStake) && (confirmationCount >= network.Consensus.CoinbaseMaturity))
{
	yield return new UnspentOutputReference
	{
		Account = this,
		Address = address,
		Transaction = transactionData
	};
}
```

Reply:
Done